gobblin-modules/gobblin-azkaban/build.gradle

```diff
@@ -38,6 +38,7 @@ dependencies {
   compile externalDependency.httpclient
   compile externalDependency.httpmime
   compile externalDependency.jodaTime
+  compile externalDependency.log4j
```

Comment:
I think we should avoid this. Why do we need to add this dependency?

Reply:
The log4j dependency of this module was explicitly removed in https://github.com/apache/gobblin/pull/3081/files. This module depends on log4j-over-slf4j module, and you should be able to use the log4j APIs via that module.

Reply:
That was unintentional, removed this dependency.